build.settings.ps1

```diff
@@ -54,6 +54,29 @@ Properties {
         'My Custom Host with scriptanalyzer support'
     )
 
+    # ------------------- Testing properties ----------------------------------
+
+    # To control the build with code coverage, the $CodeCoverageStop variable can be used.
+    # The supported values for this variable are $true, $false and $null. $true enables
+    # code coverage, with the $CodeCoveragePercentage variable to reach (info below).
+    # $false disables code coverage entirely, while $null enables code coverage, but will
+    # only report on coverage status and not stop the build.
+    [System.Diagnostics.CodeAnalysis.SuppressMessage('PSUseDeclaredVarsMoreThanAssigments', '')]
+    $CodeCoverageStop = $null
```

Comment:
Are we sure this should `$null` instead of `$false`?  I'm concerned about two things.  Performance - does this slow down the test process noticeably?  And does it generate tons of output that I might not care about at the moment?   IOW when I'm fixing a bug I want to know that my new test(s) passed without having to wade through a bunch of output.   It is somewhat infrequent when I go in to look at code coverage.  Don't get me wrong, it is nice data to be able to get your hands on.  Just not sure I need to see all that output all the time.  This is probably worthy of a broader discussion and perhaps stick time using it.  If you want, check it in this way and we can see how we like it or not.